libs/sdk-core/src/binding.rs

```diff
@@ -415,12 +421,13 @@ impl log::Log for BindingLogger {
     fn flush(&self) {}
 }
 
-fn get_breez_services() -> Result<&'static BreezServices> {
-    let n = BREEZ_SERVICES_INSTANCE.get();
-    match n {
-        Some(a) => Ok(a),
-        None => Err(anyhow!("Node service was not initialized")),
-    }
+fn get_breez_services() -> Result<Arc<BreezServices>> {
+    block_on(async move {
```

Comment:
Isn't using block_on is an issue here given the fact that all other methods that use `get_breez_services` are already being invoked inside a blocking code?

Reply:
Done